src/store/audio.ts

```diff
@@ -128,6 +128,197 @@ export const REGISTER_AUDIO_ITEM = "REGISTER_AUDIO_ITEM";
+  SET_AUDIO_CHARACTER_INDEX(payload: {
+    audioKey: string;
+    characterIndex: number;
+  }): void;
+  CHANGE_CHARACTER_INDEX(payload: {
+    audioKey: string;
+    characterIndex: number;
+  }): void;
+  REMOVE_ALL_AUDIO_ITEM(): void;
+  REGISTER_AUDIO_ITEM(payload: {
+    audioItem: AudioItem;
+    prevAudioKey: string | undefined;
+  }): string;
+  SET_ACTIVE_AUDIO_KEY(payload: { audioKey?: string }): void;
+  GET_AUDIO_CACHE(payload: { audioKey: string }): void;
+  SET_ACCENT_PHRASES(payload: {
+    audioKey: string;
+    accentPhrases: AccentPhrase[];
+  }): void;
+  SET_SINGLE_ACCENT_PHRASE(payload: {
+    audioKey: string;
+    accentPhraseIndex: number;
+    accentPhrases: AccentPhrase[];
+    popUntilPause: boolean;
+  }): void;
+  SET_AUDIO_QUERY(payload: { audioKey: string; audioQuery: AudioQuery }): void;
+  FETCH_ACCENT_PHRASES(payload: { audioKey: string }): void;
+  FETCH_SINGLE_ACCENT_PHRASE(payload: {
+    audioKey: string;
+    newPronunciation: string;
+    accentPhraseIndex: number;
+    popUntilPause: boolean;
+  }): void;
+  FETCH_MORA_DATA(payload: {
+    audioKey: string;
+    changeIndexes?: number[];
+  }): void;
+  FETCH_AUDIO_QUERY(payload: { audioKey: string }): void;
+  SET_AUDIO_ACCENT(payload: {
+    audioKey: string;
+    accentPhraseIndex: number;
+    accent: number;
+  }): void;
+  TOGGLE_ACCENT_PHRASE_SPLIT(
+    payload:
+      | {
+          audioKey: string;
+          accentPhraseIndex: number;
+          isPause: false;
+          moraIndex: number;
+        }
+      | {
+          audioKey: string;
+          accentPhraseIndex: number;
+          isPause: true;
+          moraIndex: undefined;
+        }
+  ): void;
+  CHANGE_ACCENT_PHRASE_SPLIT(payload: {
+    audioKey: string;
+    accentPhraseIndex: number;
+    moraIndex: number | null;
+    isPause: boolean;
+  }): void;
+  SET_AUDIO_MORA_DATA(payload: {
+    audioKey: string;
+    accentPhraseIndex: number;
+    moraIndex: number;
+    pitch: number;
+  }): void;
+  GENERATE_AUDIO(payload: { audioKey: string }): Blob | null;
+  GENERATE_AND_SAVE_AUDIO(payload: {
+    audioKey: string;
+    filePath?: string;
+    encoding?: EncodingType;
+  }): SaveResultObject;
+  GENERATE_AND_SAVE_ALL_AUDIO(payload: {
+    dirPath?: string;
+    encoding: EncodingType;
+  }): SaveResultObject[];
+  IMPORT_FROM_FILE(payload: { filePath?: string }): string[] | void;
+  PLAY_AUDIO(payload: { audioKey: string }): boolean;
+  STOP_AUDIO(payload: { audioKey: string }): void;
+  PLAY_CONTINUOUSLY_AUDIO(): void;
+  STOP_CONTINUOUSLY_AUDIO(): void;
+  PUT_TEXTS(payload: {
+    texts: string[];
+    characterIndex: number | undefined;
+    prevAudioKey: string | undefined;
+  }): void[];
+  OPEN_TEXT_EDIT_CONTEXT_MENU(): void;
+  DETECTED_ENGINE_ERROR(): void;
+  RESTART_ENGINE(): void;
+  CHECK_FILE_EXISTS(payload: { file: string }): boolean;
+};
+
+export type AudioCommandGetters = Record<string, never>;
```

Comment:
メモ: 以下でないとうまくいかない可能性があるので、置き換えることになるかもしれない...
```suggestion
// eslint-disable-next-line @typescript-eslint/ban-types
export type AudioCommandGetters = {};
```